columnphysics/icepack_snow.F90

```diff
@@ -1194,6 +1194,7 @@ subroutine drain_snow (nslyr, vsnon, aicen, &
       else
          sliq = meltsliq  ! computed in thickness_changes
       endif
+!tcx??
       meltsliq = meltsliq
```

Comment:
This line can be commented out.  This section of the code was rewritten a bit and corrected, and this line was the logical culmination of that process.

Reply:
> > Do we want a PR for each feature refactor or just one (or a few) PRs to cover multiple feature refactors? Would like to get some feedback before moving forward.
> 
> If I were doing it, I think I would probably do it that way:
> 
>     * 1 PR per module (so 1 per file)
> 
>     * In each PR, one commit per subroutine: removing `l_*` local variable, using `argcheck` to validate arguments, and adjusting calls to other subroutines if need be.

I don't think 1 PR per module will work well.  The optional argument checking will depend on the underlying implementation and that needs to be coordinated.  Removing the l_ variables might result in further optimizations down the calling tree.  So maybe 1 PR per feature.  I also understand about general cleanup (formatting) with updating features.  I agree, it's just hard not to fix things when I see an opportunity, and I'm worried I may not circle back if I don't.  I'll try to be more careful.